spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/SparkSQLProperties.java

```diff
@@ -47,4 +47,8 @@ private SparkSQLProperties() {}
   public static final String PRESERVE_DATA_GROUPING =
       "spark.sql.iceberg.planning.preserve-data-grouping";
   public static final boolean PRESERVE_DATA_GROUPING_DEFAULT = false;
+
+  // Influence write distribution mode using spark SQL config
```

Comment:
Nit could the comment just be `Control write distribution mode` similar pattern to the existing configs, it's explanatory from the class itself that it's for SparkSQL properties.